test/test_scanner.py

```diff
@@ -44,23 +44,20 @@ def setup_class(cls):
         else:
             print("Skip NVD database updates.")
         # Instantiate a scanner
-        cls.scanner = Scanner(cls.cvedb)
+        cls.cve_scanner = CVEScanner().__enter__()
+        cls.scanner = VersionScanner(cls.cve_scanner)
         # temp dir for mapping tests
         cls.mapping_test_dir = tempfile.mkdtemp(prefix="mapping-test-")
         # temp dir for tests that require downloads
         cls.package_test_dir = tempfile.mkdtemp(prefix="package_test-")
 
     @classmethod
     def teardown_class(cls):
+        cls.cve_scanner.cursor.close()
```

Comment:
Why are we not calling `__exit__`? Shouldn't it call `.close()` for cursor and connection?

Reply:
Fixed it.